web/js/containers/share.js

```diff
@@ -213,10 +210,18 @@ class ShareLinkContainer extends Component {
 }
 
 function mapStateToProps(state) {
-  const { config } = state;
+  const { compare, config, date } = state;
+  const { active, isCompareA } = compare;
+  const { selected, selectedB } = date;
+
+  // selectedDate for shared permalink
+  const selectedDate = !active && !isCompareA
+    ? selectedB
+    : selected;
```

Comment:
Seeing this made me realize that I didn't take the right "selected" date into consideration when doing the coverage facet.  Maybe this is minor but, I wonder if it's worth adding a Redux selector to get the selected date which then holds this logic of determining the active compare state first and then returning `state.date.selected` or `state.date.selectedB` accordingly?  If so I can implement when working [this issue](https://github.com/nasa-gibs/worldview/issues/2973).

Reply:
That's a great idea. This compare A/B check pattern does repeat itself around the app in a few places.

Reply:
Would be a good idea to use the `getSelectedDate` selector here now